doc/1/plugins/guides/events/core-hotelclerk-remove-room-for-customer/index.md

```diff
@@ -0,0 +1,17 @@
+---
+code: true
+type: page
+title: core:hotelClerk:removeRoomForCustomer
+---
+
+# core:hotelClerk:removeRoomForCustomer
+
+
+| Arguments        | Type              | Description                                                                                  |
+| -----------------| ----------------- | -------------------------------------------------------------------------------------------- |
+| `RequestContest` | <pre>object</pre> | [requestContext](https://docs.kuzzle.io/core/1/protocols/api/context/requestcontext/) object |
+| `room`           | <pre>object</pre> | Joined room information in Koncorde format                                                                     |
```

Comment:
Koncorde doesn't know what rooms are, only the HotelClerk does.

Besides, you cannot document that object as-is, you have to explain what can be found inside (of interest for the user)

Namely:
* index
* collection

Aaaaaand... that's all: I don't think we should document the customers and channels properties. 

You should probably inject the roomId in the provided payload because, otherwise, that object is useless to users (you'll have to duplicate the object to prevent mutations on the original object).